src/store/index.js

```diff
@@ -56,7 +56,6 @@ const saveStateToLocalStorage = (state) => {
 export default function configureStore(initialState) {
   const store = createStore(reducer, initialState, applyMiddleware(thunk));
   updateGraphOnChange(store);
-
```

Comment:
Unless Prettier is doing this automatically I think ideally try to retain the original whitespace as much as possible, as often newlines are used to help separate logical groups or emphasise certain lines, the smaller change set can also make reviews a bit easier too.

Reply:
I agree with @bru5 , and on top of that this shouldn't be commited as part of this PR. @rashidakanchwala we can go through your set up to ensure these unrelated changes in files that are unrelated to this PR doesn't get committed